extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -234,6 +234,14 @@ export class GoogleSubscriptionsPlatform {
     //TODO: implement this
     return Promise.resolve(false);
   }
+
+  /** @override */
+  decorateUI(element, action, options) {
+    if (action === 'login') {
```

Comment:
It's the other way around: only `subscribe` can be decorated.

Reply:
Done